### PR TITLE
Add lacunarity feature

### PR DESCRIPTION
**Pull request by CMFiltenborg**
Hi @bouweandela, this will not work without setting the .canny_edge on the window...

I could add my image.py to this PR but that is up to you...

That file contains some other stuff/bug fixes I think.

### CODE REVIEW

**Comment by bouweandela** [comment]
@CMFiltenborg Yes, it would be good if you could add your image.py too.

**Comment by CMFiltenborg** [comment]
@bouweandela  added it.

File: satsense/image.py [hunk 1]
```diff
@@ -74,7 +93,7 @@ def shallow_copy_range(self, x_range, y_range, pad=True):
             self._normalized_image = normalize_image(
                 self.raw, self.bands, **self._normalization_parameters)
 
-        im._normalized_image = self._normalized_image[x_range, y_range]
+            im._normalized_image = self._normalized_image[x_range, y_range]
```
Comment (bouweandela) [inline satsense/image.py]: Why is this changed? Was it a bug?

File: satsense/image.py [hunk 2]
```diff
@@ -61,6 +65,21 @@ def gray_ubyte(self):
             self._gray_ubyte_image = get_gray_ubyte_image(self.rgb, RGB)
         return self._gray_ubyte_image
 
+    @property
+    def canny_edged(self):
+        try:
+            if self._canny_edge_image is None:
+                grayscale = np.copy(self.grayscale)
```
Comment (bouweandela) [inline satsense/image.py]: Why the copy? It looks like [`skimage.filters.rank.equalize`](http://scikit-image.org/docs/dev/api/skimage.filters.rank.html#skimage.filters.rank.equalize) creates a new output array, the way you use it.

Comment (CMFiltenborg) [reply satsense/image.py]: Hmm I guess this is leftover from debugging

File: satsense/image.py (hunk 1 above)
Comment (CMFiltenborg) [reply satsense/image.py]: Yes, I think I have had errors on this line. Changed this some time ago.
